test/unit/test-dom.js

```diff
@@ -747,6 +747,38 @@ describes.sandboxed('DOM', {}, env => {
         });
       });
     });
+
+    it('should yield body asap even if doc is not complete', () => {
+      return new Promise((resolve, reject) => {
+        const doc = {
+          readyState: 'loading',
+          body: null,
+          documentElement: {
+            ownerDocument: {
+              defaultView: {
+                setInterval() {
```

Comment:
Could just `setInterval: window.setInterval`?

Reply:
I tried. It gets "invalid invocation". I guess it gets the wrong `this`.